src/modules/data/Height.ts

```diff
@@ -64,4 +64,15 @@ export class Height
     {
         return Utils.UInt64ToString(this.value);
     }
+
+    /**
+     * Converts this object to its JSON representation
```

Comment:
`     * Converts this object to it's JSON representation`

Reply:
In this case, `its` is correct.
https://www.grammarly.com/blog/its-vs-its
And yes, it's (!) weird to me too ;)

Reply:
Oh, TIL